integration-tests/set-embrace-config.js

```diff
@@ -119,6 +132,13 @@
         enable_automatic_activity_capture: false,
       };
     }
+
+    if (config.enable_network_span_forwarding || config.disabled_url_patterns) {
+      androidConfig.sdk_config.networking = {
+        enable_network_span_forwarding: true,
```

Comment:
since the check is an `||` should handle one or the other of the configs being undefined here

Reply:
done